tools/reactive-controllers/src/delegate-controllers/SpectrumTwoDownStateController.ts

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import type { ReactiveController } from 'lit';
+import { SpectrumElement } from '@spectrum-web-components/base';
+
+export class SpectrumTwoDownStateController implements ReactiveController {
+    private host: SpectrumElement;
+
+    constructor(host: SpectrumElement) {
+        this.host = host;
+        this.host.addController(this);
+    }
+
+    public hostConnected(): void {
+        if (this.host.spectrumConfig?.downstate?.includes('spectrum-two')) {
```

Comment:
you will not be able to access spectrumConfig from `this.host` since you can creating an instance of SpectrumElement and static property will not be accessible.
I have updated `spectrumConfig` to be public in `SpectrumElement`
```js
 public spectrumConfig: SpectrumConfig = {};
 ```